WowPacketParser/App.config

```diff
@@ -268,6 +268,13 @@
         -->
         <add key="SaveTempSpawns" value="false"/>
 
+		<!--
+             Option:      SaveExistingSpawns
+             Description: Spawn spawns which are already existing in db (compared by entry, position) or comment them in sql
+             Default:     "false" (comment existing spawns in sql)
+        -->
+		<add key="SaveExistingSpawns" value="false"/>
```

Comment:
@mdX7 prolly just one thing, default should be True, thats the old behaviour. Forgot to change that.

Reply:
Im going with default false, makes it easier for new people using wpp